Doc/whatsnew/3.14.rst

```diff
@@ -124,23 +124,7 @@ Here's a simple example that inspects object types in a running Python process:
      # Create a temporary script
      with tempfile.NamedTemporaryFile(mode='w', suffix='.py', delete=False) as f:
          script_path = f.name
-         f.write("""
-     import gc
-     import collections
-
-     # Collect all objects managed by the garbage collector
-     gc.collect()
-
-     # Count objects by type
-     type_counts = collections.Counter(type(obj).__name__
-                                       for obj in gc.get_objects())
-
-     # Print the most common types
-     print("Most common object types in process:")
-     for type_name, count in type_counts.most_common(10):
-         print(f"  {type_name}: {count}")
-     """)
-
+         f.write("import my_debugger; my_debugger.connect(HOST_PROCESS)")
```

Comment:
This is, unfortunately, going to raise the question of how does `HOST_PROCESS` get set. Presumably we'd substitute in our own PID when writing out the script?

```suggestion
         f.write(f"import my_debugger; my_debugger.connect({os.getpid()})")
```

Reply:
Agreed, although it looks a bit weird that you schedule on yourself in the example